AI_Graphics/AI_Graphics/Inspector/PresetInspector.cs

```diff
@@ -42,8 +42,10 @@ internal static void Draw(PresetManager presetManager)
             _nameToSave = GUILayout.TextField(_nameToSave);
             bool isValidFileName = (0 != _nameToSave.Length && 256 >= _nameToSave.Length);
             bool isCue = (_nameCue == _nameToSave);
-            if (GUILayout.Button("Save", GUILayout.ExpandWidth(false)) && isValidFileName && !isCue )
+            if (GUILayout.Button("Save", GUILayout.ExpandWidth(false)) && isValidFileName && !isCue) {
                 presetManager.Save(_nameToSave);
+                _presetIndexCurrent = Array.IndexOf(presetManager.PresetNames, presetManager.CurrentPreset);
```

Comment:
Do you know why we would need this? The next Draw() call will do exactly this before the _presetIndexCurrent is used right?

Reply:
Oh hell, it's supposed to be _presetIndexOld -.-

Reply:
Fixed with commit f8772a4